User story 2

jamesclarence opened the pull request:
A user is able to sign in and sign out, and the navigation changes to indicate a User is signed out. The `bootstrap-sass` gem is updated to the most recent version.

jamesclarence committed:
add sign in link; update about route

jamesclarence committed:
errors adding bootstrap styling

jamesclarence committed:
updated bootstrap-sass gem

jamesclarence committed:
add user stories to README

xander-miller commented:
Looks good to merge to me. Could you build a seed file next with an admin user and some dummy data so it is easier to poke and click around?

![](http://i.giphy.com/CE2JyzhfSycrm.gif)

jamesclarence commented:
I've added the `faker` gem to the `user-story-3` branch to start seeding. I've added some users, but still need to add an `admin` role. https://github.com/jamesclarence/blocitoff/tree/user-story-3
